### PR TITLE
Upgrade to Angular 14. Update package versions

### PR DESCRIPTION
**Pull request by ryan-henness-trimble**
## Description

Upgrade the `angular-workspace` to use Angular 14 (specifically 14.1.1). Update the Angular and React packages to use new naming convention.

References #614 

## Type of change

- [x] Framework target package output

### CODE REVIEW

**Comment by ryan-henness-trimble** [comment]
@leohaughton it won't let me add the account as a reviewer

File: angular-workspace/angular.json [hunk 1]
```diff
@@ -137,6 +137,5 @@
         }
       }
     }
-  },
-  "defaultProject": "@trimble-oss/modus-angular-components"
```
Comment (ryan-henness-trimble) [inline angular-workspace/angular.json:141]: This seemed to me removed automatically during the Angular upgrade?

File: angular-workspace/projects/trimble-oss/modus-angular-components/package.json [hunk 2]
```diff
@@ -12,9 +12,9 @@
     "url": "https://github.com/trimble-oss/modus-web-components"
   },
   "peerDependencies": {
-    "@angular/common": "^13.3.0",
-    "@angular/core": "^13.3.0",
-    "@trimble-oss/modus-web-components": "^0.1.9"
```
Comment (ryan-henness-trimble) [inline angular-workspace/projects/trimble-oss/modus-angular-components/package.json:17]: Note: removed caret ranges. Seems like we would want the exact version here?

**Comment by trimbleo** [comment]
Oops I responded using my personal account, which is why you couldnt assign it to me.

**Comment by trimbleo** [comment]
Looks good to me.

**Pull request merged**